TACC/Core-CMS-Custom
# Task/WP-762--Use-Core-Component-Buttons

## Overview
Some components use Reactstrap Button, others use Core-Components Button. Check across APCD React client to make sure we’re using Core-Components button so that style changes from Core-CMS and Core-Styles will be picked up
…

## Related

[WP-762](https://tacc-main.atlassian.net/browse/WP-762)


## Changes
- Removed Reactstrap Button in favor of Core-Components button
- Requires attribute submit instead of type submit to ensure submission action on Formik component
…

## Testing

1. Navigate to the Registration Form
2. Navigate to Edit Record Modal in User List and ensure component buttons match new styling

## UI
### Before
#### Edit User Modal
![image](https://github.com/user-attachments/assets/dd34ae3f-2732-4a43-b470-9728c8a030a5)

### After
#### Edit User Modal
![image](https://github.com/user-attachments/assets/a2efefdf-a053-4ca9-93b6-b5f2d6710fa4)

…


## Notes
- For registration forms the styling does not seem to change, but for edit record modal it does
…
